contracts/AuctionBase.sol

```diff
@@ -49,24 +55,29 @@ contract AuctionBase is Pausable {
     uint256 tokenId,
     address seller,
     uint256 bidIncrement,
-    uint64 duration,
-    uint64 startedAt,
-    bool isActive,
+    uint256 duration,
+    uint256 startedAt,
+    uint256 startBlock,
+    AuctionStatus status,
     uint256 highestBid,
-    address highestBidder
+    address highestBidder,
+    bool cancelled
   ) {
-    Auction storage auction = auctions[_id];
+    Auction memory _auction = auctions[_id];
+    AuctionStatus _status = _getAuctionStatus(_id);
```

Comment:
You don't have to specify `memory` or `storage` here?

Reply:
Nope

Reply:
I don't think so